optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -514,7 +515,9 @@ private XYPlot createScalabilityPlot(List<XYSeries> seriesList,
             String yAxisLabel, NumberFormat yAxisNumberFormat) {
         NumberAxis xAxis = new NumberAxis(xAxisLabel);
         if (useLogarithmicProblemScale(seriesList)) {
-            xAxis = new LogarithmicAxis(xAxis.getLabel() + " (logarithmic)");
+            LogarithmicAxis xLogAxis = new LogarithmicAxis(xAxis.getLabel() + " (logarithmic)");
+            xLogAxis.setAllowNegativesFlag(true);
+            xAxis = xLogAxis;
```

Comment:
the `setAllowNegativesFlag()` is a method of LogarithmAxis, that's why I found this way better than doing `((LogarithmicAxis)xAxis).setAllowNegativesFlag(true);`